datasets.md

```diff
@@ -0,0 +1,87 @@
+
+### Datasets in dataset_definitions.yml
+* `imagenet_1000_classes` used for evaluation models trained on ILSVRC 2012 dataset for 1000 classes. (model examples: alexnet, colorization-v2)
+* `imagenet_1000_classes_2015` used for evaluation models trained on ILSVRC 2015 dataset for 1000 classes. (model examples: se-resnet-152, se-resnext-50)
+* `imagenet_1001_classes` used for evaluation models trained on ILSVRC 2012 dataset for 1001 classes (background label + original labels). (model examples: googlenet-v2-tf, resnet-50-tf)
+
+## [Common Objects in Context (COCO)](http://cocodataset.org/#home)
+
+### How download dataset
+
+To download COCO dataset, you need to follow the steps below:
+1. Go to the [COCO](http://cocodataset.org/#home) website
+2. Go to the `Dataset-> Download` tab
+3. Select [`2017 Val images`](http://images.cocodataset.org/zips/val2017.zip) and [`2017 Train/Val annotations`](http://images.cocodataset.org/annotations/annotations_trainval2017.zip) and download these archives
+4. Unpack archives
+
+### Structure of directory with dataset
+
+This section describes what structure of directory dataset must have to run the Accuracy Checker tool.
+* `<DATASET_DIR>` - root directory
+    * `val2017` - directory containing the COCO 2017 validation images
+    * `instances_val2017.json` - annotation file which used for object detection and instance segmentation tasks
+    * `person_keypoints_val2017.json` - annotation file which used for human pose estimation tasks
+
+### Datasets in dataset_definitions.yml
+* `ms_coco_mask_rcnn` used for evaluation models trained on COCO dataset for object detection and instance segmentation tasks. Background label + label map with 80 public available object categories are used. Annotations are saved in image id ascend order.
+* `ms_coco_detection_91_classes` used for evaluation models trained on COCO dataset for object detection tasks. Background label + original label map with 91 object categories are used (public available 80 categories). Annotations are saved in image id ascend order. (model examples: faster_rcnn_resnet50_coco, ssd_resnet50_v1_fpn_coco)
+* `ms_coco_detection_80_class_with_background` used for evaluation models trained on COCO dataset for object detection tasks. Background label + label map with 80 public available object categories are used. Annotations are saved in image id ascend order. (model examples: faster-rcnn-resnet101-coco-sparse-60-0001, ssd-resnet34-1200-onnx)
+* `ms_coco_detection_80_class_without_background` used for evaluation models trained on COCO dataset for object detection tasks. Label map with 80 public available object categories is used. Annotations are saved in image id ascend order. (model examples: ctdet_coco_dlav0_384, pelee-coco)
+* `ms_coco_keypoints` used for evaluation models trained on COCO dataset for human pose estimation tasks. Each annotation stores multiple keypoints for one image. (model examples: human-pose-estimation-0001, openpose-pose-caffe)
```

Comment:
there is no openpose-pose-caffe model in OMZ
```suggestion
* `ms_coco_keypoints` used for evaluation models trained on COCO dataset for human pose estimation tasks. Each annotation stores multiple keypoints for one image. (model examples: human-pose-estimation-0001)
```

Reply:
It is my mistake. Model has been removed